clarity-ext-scripts/clarity_ext_scripts/covid/rtpcr_analyse_execution.py

```diff
@@ -71,6 +70,13 @@ def execute(self):
         self.context.current_step.udf_map.force("rtPCR Passed", rt_pcr_passed)
         self.context.update(self.context.current_step)
 
+    def _get_control_name(self, key):
```

Comment:
Lets put this in controls later, because we might need it in other places as well, but I'll merge this now.